src/rules/prefer-in-document.js

```diff
@@ -184,6 +247,12 @@ export const create = (context) => {
         context,
         node.object.object.arguments[0].name
       );
+
+      // Not an RTL query
+      if (!queryNode || queryNode.type !== "CallExpression") {
```

Comment:
I think we can use optional chaining here:

```suggestion
      if (queryNode?.type !== "CallExpression") {
```